src/security/PeerConnector.cc

```diff
@@ -763,16 +763,23 @@ Security::PeerConnector::resumeNegotiation()
     auto lastError = suspendedError_; // may be reset below
```

Comment:
PR changes make this comment stale, at least for some reasonable definitions of "reset" and "below".
```suggestion
    auto lastError = suspendedError_;
```

Reply:
commit 3b2d267335